models/surveys.go

```diff
@@ -144,6 +145,11 @@ func (api *API) AllSurveys(w http.ResponseWriter, r *http.Request) {
 func (api *API) GetSurvey(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	id := vars["surveyId"]
+
+	if !isUuid(id, "Survey", w) {
```

Comment:
When listing the field in error should be reporting the actual paramname - i.e. `surveyId` instead of `Survey`